### PR TITLE
King Unit Tests

### PR DESCRIPTION
**Pull request by Jason-mcm**
King class unit tests

### CODE REVIEW

**Review by natalinamateaki — CHANGES_REQUESTED** [review]
Looks great! I think one more thing that would be good to test for is to ensure that possibleMoves() doesn't add a square in it's path that has a piece with the same player Id.

File: test/rook-test.js
Comment (natalinamateaki) [inline test/rook-test.js:1]: Looks great, only thing I found is that on line 148, I think you meant to write "board.board[0][1] = rook3".

**Review by natalinamateaki — CHANGES_REQUESTED** [review]
On line 148 in rook-test.js, I think you meant to write "board.board[0][1] = rook3".

**Review by natalinamateaki — APPROVED** [review]
Looks great! Good work!

**Review by jbeBan — APPROVED** [review]
💯

**Pull request merged**